ui/src/components/queries/ModelListModalQuery.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 // import { get } from 'lodash';
-
 import { useDataContext } from '@overture-stack/arranger-components/dist/DataContext';
```

Comment:
whenever avoidable, please refrain from deep imports (e.g. reaching into files withing the dist folder).
https://hackmd.io/@dzearing/ry1jTjCKd

```suggestion
import { useArrangerData as useDataContext } from '@overture-stack/arranger-components';
```

Reply:
Oh yeah this one really isn't necessary, will remove

Reply:
Updated these imports across the app